app/Insightly/Resources/InsightlyProjectResource.php

```diff
@@ -19,14 +23,48 @@ public function __construct(
 
     public function create(Integration $integration): int
     {
-        return 0;
+        $request = new Request(
+            'POST',
+            $this->path,
+            [],
+            JSON::encode(
+                (new ProjectSerializer($this->insightlyClient->getPipelines()))
```

Comment:
Would it make sense to move this project serializer construction to a method? Feels like there's too much knowledge required here on how to construct such a serializer, knowledge that's repeated further down as well.

Reply:
It could even be injected at construction time I think.

Reply:
We could do that, although this is all Insightly-related code.